CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+Possible log types are `[add]`, `[bug]`, `[change]` and `[info]`.
```

Comment:
I'm saying that you shouldn't just filter through the changelog to see the breaking changes. You should take a look at all changes, as they might affect you in some way or another. So the "breaking" flag doesn't really tell you much :)